tools/utils/_functions.sh

```diff
@@ -34,7 +34,7 @@ PACKAGE_DIR="${BASE_DIR}/${PACKAGE_DIR_REL}"
 # The filename of the currently running script
 SCRIPT_NAME=$(basename "$0")
 # The absolute path to the currently running script (required to allow restarting with different permissions
-SCRIPT_PATH="${DEV_TOOL_DIR}/${SCRIPT_NAME}"
+SCRIPT_PATH="${BASE_DIR[0]}/tools/${SCRIPT_NAME}"
```

Comment:
> Changed SCRIPT_PATH calculation, because it otherwise looked for the script at the wrong place. Tested the other script, should work as expected, but it is not reall nice to hard code tools folder name here. Any better ideas?

Maybe change the variables above?
```
DEV_TOOL_DIR=$(pwd)
cd ../..
```
to
```
cd ..
DEV_TOOL_DIR=$(pwd)
cd ..
```
or something similar? Then we can rely on the fact that `DEV_TOOL_DIR` is really the dev tool dir, in case we copy more dev tools from integrat-cms...

and keep this  line as it was?

```suggestion
SCRIPT_PATH="${DEV_TOOL_DIR}/${SCRIPT_NAME}"
```